src/configparser.cpp

```diff
@@ -35,7 +34,7 @@ void ConfigParser::handle_action(const std::string& action,
 			throw ConfigHandlerException(
 				ActionHandlerStatus::TOO_FEW_PARAMS);
 		}
-		if (!this->parse(utils::resolve_relative(included_files.back(), utils::resolve_tilde(params[0]))))
+		if (!this->parse(utils::resolve_relative(included_files_stack.top(), utils::resolve_tilde(params[0])), false))
```

Comment:
This line was hard for me to understand because of all the nesting. Can you split it into multiple lines, with nicely named intermediate variables?

Reply:
Attempted fix in 71ab2ab

Reply:
That's much better, thank you!